.github/workflows/docs.yml

```diff
@@ -0,0 +1,55 @@
+name: "Docs build and deploy"
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+defaults:
+  run:
+    shell: bash -l {0}
+
+jobs:
+  build-and-deploy:
+    name: "Docs - build and deploy"
+    runs-on: ubuntu-latest
+    env:
+      OE_LICENSE: ${{ github.workspace }}/oe_license.txt
+
+    steps:
+      - uses: actions/checkout@v3
+      
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+            auto-update-conda: true
+            use-mamba: true
+            python-version: 3.9
+            miniforge-variant: Mambaforge
+            environment-file: devtools/conda-envs/docs.yml
+            activate-environment: fah-alchemy-test
+
+      - name: Decrypt OpenEye license
+        if: ${{ !github.event.pull_request.head.repo.fork }}
+        env:
+          OE_LICENSE_TEXT: ${{ secrets.OE_LICENSE }}
+        run: |
+          echo "${OE_LICENSE_TEXT}" > ${OE_LICENSE}
+          python -c "import openeye; assert openeye.oechem.OEChemIsLicensed(), 'OpenEye license checks failed!'"
+
+      - name: "Install"
+        run: python setup.py develop --no-deps
+
+      - name: "Build docs"
+        run: |
+          cd docs
+          make html
+
+      - name: Deploy 🚀
+        if: github.event_name != 'pull_request'
+        uses: JamesIves/github-pages-deploy-action@3.7.1
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          BRANCH: gh-pages # The branch the action should deploy to.
+          FOLDER: docs/_build/html # The folder the action should deploy.
```

Comment:
I take it the `if` statement will only actually deploy on merge into `main`? If so, that is the desired outcome I think.

Reply:
Yeah any push to main should trigger deploy.